packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/AutoLayoutTable.jsx

```diff
@@ -1,6 +1,32 @@
+/* eslint-disable react/forbid-dom-props */
 import React from 'react';
 import Table from 'terra-table';
 
+const longText = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
+
+const TruncatedText = () => (
+  <div style={{
```

Comment:
Shouldn't we be able to add styles on the td element directly?  Do we need consumers to create nested div containers?  It looks like the changes might impact our desired layout for the cell.

Reply:
The way I was able to implement the truncation involves absolutely positioned wrapper, which can't be <td> as it would mess with its width and needs to be placed inside of <td>. Possibly, if some other css implementation exists, that would be possible to apply the styles to <td>, however in that case my concern with adding styles to <td> element is that consumers can override our styles for <td>, which is also not good.

Reply:
I updated the `<td>` element styling [here](https://github.com/cerner/terra-framework/pull/2131/commits/78c887c9fb73d23f947dea7abe14e3549fc86537), but please let me know if you know a better way to do that.